src/app/profile/shared/profile.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Lot } from '@app/shared/models/lot.model';
+import { environment } from '@env/environment';
+
+import { ProfileModule } from '../profile.module';
+
+@Injectable({ providedIn: ProfileModule })
+export class ProfileService {
+  private backUrl = `${environment.apiUrl}/api/lots`;
+
+  createLot(lotData: Lot) {
+    return this.http.post(this.backUrl, lotData);
```

Comment:
_POST_ **_/api/lots_** is protected route, you should be authorized to make this request, otherwise you will get 401 response

Reply:
done